packages/lesswrong/components/sequences/LargeSequencesItem.tsx

```diff
@@ -123,54 +134,61 @@ export const LargeSequencesItem = ({sequence, showAuthor=false, classes}: {
+
+  const highlight = sequence.contents?.htmlHighlight || ""
 
   return <div className={classes.root} >
 
     <div className={classes.columns}>
       <div className={classes.left}>
-        <div className={classes.sequenceImage}>
-          <img className={classes.sequenceImageImg}
-            src={`https://res.cloudinary.com/${cloudinaryCloudName}/image/upload/c_fill,dpr_2.0,g_custom,h_96,q_auto,w_292/v1/${
-              sequence.gridImageId || "sequences/vnyzzznenju0hzdv6pqb.jpg"
-            }`}
-            />
-        </div>
+        <Link className={classes.imageLink} to={'/s/' + sequence._id}>
+          <div className={classes.sequenceImage}>
+            <img className={classes.sequenceImageImg}
+              src={`https://res.cloudinary.com/${cloudinaryCloudName}/image/upload/c_fill,dpr_2.0,g_custom,h_96,q_auto,w_292/v1/${
+                sequence.gridImageId || "sequences/vnyzzznenju0hzdv6pqb.jpg"
+              }`}
+              />
+          </div>
+        </Link>
         <div className={classes.text}>
           <div className={classes.titleAndAuthor}>
-            <Link to={'/s/' + sequence._id} className={classes.title}>{sequence.title}</Link>
+            <Link  to={'/s/' + sequence._id} className={classes.title}>{sequence.title}</Link>
           { showAuthor && sequence.user &&
             <div className={classes.author}>
               by <UsersName user={sequence.user} />
             </div>}
           </div>
-          <ContentStyles contentType="postHighlight" className={classes.description}>
+          {(highlight.length > 0) && <ContentStyles contentType="postHighlight" className={classes.description}>
             <ContentItemTruncated
               maxLengthWords={100}
               graceWords={20}
               rawWordCount={sequence.contents?.wordCount || 0}
               expanded={expanded}
               getTruncatedSuffix={() => null}
-              dangerouslySetInnerHTML={{__html: sequence.contents?.htmlHighlight || ""}}
+              dangerouslySetInnerHTML={{__html: highlight}}
               description={`sequence ${sequence._id}`}
             />
-          </ContentStyles>
+          </ContentStyles>}
+          <LWTooltip title={<div> ({totalWordcount.toLocaleString("en-US")} words)</div>}>
+            <div className={classes.wordcount}>{Math.round(totalWordcount / 300)} min read</div>
+          </LWTooltip>
         </div>
       </div>
       <div className={classes.right}>
-        {sequence.chapters?.map((chapter) => <span key={chapter._id}>
-            {chapter.posts?.map(post => <SequencesSmallPostLink 
-                                          key={chapter._id + post._id} 
-                                          post={post}
-                                        />
+        {posts.map(post => <SequencesSmallPostLink 
+                              key={sequence._id + post._id} 
+                              post={post}
+                            />
```

Comment:
Jim has [previously commented](https://github.com/ForumMagnum/ForumMagnum/pull/5105#discussion_r906367910) on this indentation style.